dnf/base.py

```diff
@@ -1405,30 +1405,42 @@ def _finalize_comps_trans(self):
         trans = self._comps_trans
         basearch = self.conf.substitutions['basearch']
 
-        def cond_check(pkg):
+        def conditional_pkg_check(pkg):
+            """
+            If package is conditional it check if transaction fulfill condition
```

Comment:
... it checks ... transaction fulfills ...

Reply:
Fixed, thanks...